docs/migration-from-4.x.x-to-5.x.x.md

```diff
@@ -1,16 +1,18 @@
 # Migration from 4.x.x to 5.x.x
 
-## Changes
+**AEproject** `v5.0.0` underwent some breaking changes.
 
-**AEproject** `v5.0.0` underwent some minor but breaking changes.
-
-Install the new AEproject version
+Install the latest AEproject version
 
 ```
 npm install -g @aeternity/aeproject
 ```
 
-### Removed from libs
+## Various Changes
+
+- **dropped commonjs support**, newly created projects will be created as esm projects, old cjs projects might not continue to work with newer aeproject versions
```

Comment:
they would definitely break because you can't import esm from cjs, since esm can have root-level await and so

Reply:
thank you, you are right and I will clarify